### PR TITLE
Updates pull request template

### PR DESCRIPTION
**Pull request by omsenjalia**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 To make our job easier, please spend time to review your application before submitting. 
 
+NOTE: Your not *required* to fill in this application.
```
Comment (sasial-dev) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Fix the grammar, and make it a comment!
```suggestion
<!-- NOTE: You're *not* required to fill in this application. -->
```

**Review by sasial-dev — CHANGES_REQUESTED** [review]
Grammar fixes! :)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (omxpro) [reply .github/PULL_REQUEST_TEMPLATE.md]: I don't think comment is a good idea as no one sees it.

**Comment by sasial-dev** [comment]
i'd also make it into a comment, still

**Comment by omxpro** [comment]
> i'd also make it into a comment, still

No one sees the comment it has no use

**Comment by sasial-dev** [comment]
That's incorrect. A comment can be seen by the __pull request creator__.

**Comment by omxpro** [comment]
> That's incorrect. A comment can be seen by the **pull request creator**.

Okay. But i really still don't wanna do it.

**Comment by sasial-dev** [comment]
why?

**Comment by mtgsquad** [comment]
> > That's incorrect. A comment can be seen by the **pull request creator**.
> 
> Okay. But i really still don't wanna do it.

You can't just say, I don't wanna do it, you have to give reasoning for everything, and stop updating the PR template for one line things, that's literally useless.

**Comment by mtgsquad** [comment]
Also, @sasial-dev, great website! Merged your PR to redirect to sasial.dev.

**Comment by VaibhavSys** [comment]
> Also, @sasial-dev, great website! Merged your PR to redirect to sasial.dev.

I went to check the website and got a 503 lol

**Pull request closed without merging**